resque-throttler.gemspec

```diff
@@ -27,5 +27,5 @@ Gem::Specification.new do |s|
   #s.add_development_dependency 'sdoc-templates-42floors'
 
   # Runtime
-  s.add_runtime_dependency 'resque', '~> 1.25'
```

Comment:
Hi @krimsonkla sorry I'm just now getting to this, didn't realize it was still in use. Would '> 1.25' be acceptable there or do you need a version < 1.25?

Reply:
I think '>= 1.25' would be fine, see also for example resque-retry, resque-scheduler, and resque-spec.